generator/src/test/java/com/scottlogic/deg/generator/cucumber/ContainingRegex.feature

```diff
@@ -280,32 +281,6 @@ Scenario: Running a 'containingRegex' request alongside a null constraint should
        | foo  |
        | null |
 
-Scenario: Running a 'containingRegex' request alongside an ofType = string should be successful
```

Comment:
Was this intentionally removed?

Reply:
Yes, this is a duplicate test found in the aValid.feature file and is redundant.